pkg/cloud/metadata_k8s.go

```diff
@@ -11,7 +27,7 @@ import (
 )
 
 const (
-	cloudInstanceIDLabel = "powervs.kubernetes.io/cloud-instance-id"
+	CloudInstanceIDLabel = "powervs.kubernetes.io/cloud-instance-id"
 	pvmInstanceIdLabel   = "powervs.kubernetes.io/pvm-instance-id"
```

Comment:
make the other one also public,

Reply:
pvmInstanceIdLabel variable is not used in public yet. should we make it public?

Reply:
As the other one is already exported, hence even this can be exported to keep that uniformity.

Reply:
fixed